pkg/cluster/api/pdapi.go

```diff
@@ -345,13 +346,23 @@ func (pc *PDClient) EvictStoreLeader(host string, retryOpt *utils.RetryOption) e
 		break
 	}
 
-	if latestStore == nil || latestStore.Status.LeaderCount == 0 {
+	if latestStore == nil {
+		return nil
+	}
+
+	// XXX: the status address in store will be something like 0.0.0.0:20180
```

Comment:
What is `XXX`?

Reply:
https://www.snellman.net/blog/archive/2017-04-17-xxx-fixme/

> The token XXX is frequently used in source code comments as a way of marking some code as needing attention.